cfg/1.11/master.yaml

```diff
@@ -544,12 +544,18 @@ groups:
     text: "Ensure that the admission control plugin NodeRestriction is set (Scored)"
     audit: "ps -ef | grep $apiserverbin | grep -v grep"
     tests:
+      bin_op: or
       test_items:
       - flag: "--enable-admission-plugins"
         compare:
           op: has
           value: "NodeRestriction"
         set: true
+      - flag: "--admission-control"
+        compare:
+          op: has
+          value: "NodeRestriction"
+        set: true
```

Comment:
I think this change is something else

Reply:
Whoops, you're right. I think it should be fixed now.